src/figuresClasses.ts

```diff
@@ -1,11 +1,88 @@
-export interface Figure {}
+export interface Figure {
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
+  getArea: () => number;
+}
+
+export class Triangle implements Figure {
+  constructor(
+    public color: 'red' | 'green' | 'blue',
+    public a: number,
+    public b: number,
+    public c: number,
+    public shape: 'triangle' = 'triangle',
+  ) {
+    if (!this.isValidTriangle()) {
+      throw new Error('insert correct data');
+    }
+  }
+
+  getArea(): number {
+    const halfPerimeter: number = (this.a + this.b + this.c) / 2;
+
+    return parseFloat(
+      Math.sqrt(
+        halfPerimeter *
+          (halfPerimeter - this.a) *
+          (halfPerimeter - this.b) *
+          (halfPerimeter - this.c),
+      ).toFixed(2),
```

Comment:
try to use `Math.floor(num * 100) / 100` instead of toFixed and parseFloat

Reply:
didn't help :(